src/main/java/stepdefs/CreateDefaultApplicationStepDefs.java

```diff
@@ -0,0 +1,53 @@
+package stepdefs;
+
+import businessobjects.User;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.cucumber.datatable.DataTable;
+import org.testng.Assert;
+import pageobject.*;
+import services.LoginService;
+import util.DataGeneration;
+
+import java.util.List;
+
+public class CreateDefaultApplicationStepDefs {
```

Comment:
as we discussed better split steps by classes that duplicate your Page Object structure
like
LoginPageSteps
HomePageSteps


Please read next comment also

Reply:
selected option from next comment